src/visualiser/visual.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @summary Finds the max distance between to numbers
+ * @param {number} NumberOne The number to add to NumerTwo
+ * @param {number} NumberTwo The number to add to NumerOne
+ * @returns {number} The result of the two numbers added togther
+ * @throws {TypeError} On invalid arguments
+ */
+function absoluteValueOfTwoNumbers(NumberOne, NumberTwo) {
```

Comment:
Really bad function name and the parameters should not start with a capital letter